### PR TITLE
WIP [es] Add concepts/storage/ephemeral-volumes.md

### PR DESCRIPTION
**Pull request by gamba47**
Adding new content to concepts/storage/ephemeral-volumes.md

Related to https://github.com/kubernetes/website/issues/13948#issuecomment-1004479727

### CODE REVIEW

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+Este documento describe los _volúmenes efímeros_ en Kubernetes. Es recomendable
+estar familiarizado con los [volúmenes](/docs/es/concepts/storage/volumes/), en 
+particular con PersistentVolumeClaim y PersistentVolume.
+
+<!-- body -->
+
+Algunas aplicaciones necesitan almacenamiento adicional pero no se preocupa si
+los datos son guardados persistentemente durante los reinicios. Por ejemplo, servicios
+de cacheo estan limitados habitualmente por el tamaño de la memoria y pueden mover de manera
```
Comment (raelga) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:21]:
```suggestion
de cacheo están limitados habitualmente por el tamaño de la memoria y pueden mover de manera
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+infrecuente datos de usuario al disco, esto es más lento que la memoria y tiene un pequeño impacto
+en el rendimiento global.
+
+Otras aplicaciones esperan algun tipo de dato de solo lectura que debe estar presente entre sus
+archivos, como los archivos de configuración con claves secretas.
+
+Los _Volúmenes efímeros_ estan diseñados para este tipo de casos. Porque los volúmenes
+siguen toda la vida del Pod y son creados y borrados junto con el
+Pod, los Pods pueden ser frenados y reiniciados sin estar limitados por
```
Comment (raelga) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:30]:
```suggestion
Pod, los Pods pueden ser parados y reiniciados sin estar limitados por
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+la ubicación y disponibilidad de algún volumen persistente. 
+
+Los volúmenes efímeros se especifican _en linea_ dentro de las especificaciones del Pod, esto
```
Comment (raelga) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:33]:
```suggestion
Los volúmenes efímeros se especifican _en línea_ dentro de las especificaciones del Pod, esto 
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+- [configMap](/docs/concepts/storage/volumes/#configmap),
+  [downwardAPI](/docs/concepts/storage/volumes/#downwardapi),
+  [secret](/docs/concepts/storage/volumes/#secret): Injectar diferentes
+  tipos de datos de Kubernetes dentro del Pod
+- [CSI ephemeral volumes](#csi-ephemeral-volumes):
+  similar al anterior tipo de volumen pero manejado por el driver 
+  [CSI drivers](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+  que específicamente [soportan esta característica](https://kubernetes-csi.github.io/docs/drivers.html)
+- [volúmenes efímeros genéricos](#generic-ephemeral-volumes), que
+  pueden ser proporcionados por todos los drivers de almacenamiento que tienen soporte para volúmenes persistentes
```
Comment (raelga) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:52]:
```suggestion
  pueden ser proporcionados por todos los controladores de almacenamiento que tienen soporte para volúmenes persistentes
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+diferentes propósitos:
+- [emptyDir](/docs/concepts/storage/volumes/#emptydir): vacío al inicio del Pod, 
+  con almacenamiento local proveniente del directorio base de kubelet ( usualmente
+  la raíz del disco) o la memoria RAM.
+- [configMap](/docs/concepts/storage/volumes/#configmap),
+  [downwardAPI](/docs/concepts/storage/volumes/#downwardapi),
+  [secret](/docs/concepts/storage/volumes/#secret): Injectar diferentes
+  tipos de datos de Kubernetes dentro del Pod
+- [CSI ephemeral volumes](#csi-ephemeral-volumes):
+  similar al anterior tipo de volumen pero manejado por el driver
```
Comment (raelga) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:48]:
```suggestion
  similar al anterior tipo de volumen pero manejado por un controlador que implemente la Container Storage Interface 
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+  [CSI drivers](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+  que específicamente [soportan esta característica](https://kubernetes-csi.github.io/docs/drivers.html)
+- [volúmenes efímeros genéricos](#generic-ephemeral-volumes), que
+  pueden ser proporcionados por todos los drivers de almacenamiento que tienen soporte para volúmenes persistentes
+
+`emptyDir`, `configMap`, `downwardAPI`, `secret` son proporcionados como
+[almacenamiento efímero local](/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage).
+Estos son administrados por kubelet en cada uno de los nodos.
+
+Los volúmenes efímeros CSI *deben* ser proporcionados por un controlador de almacenamiento
```
Comment (raelga) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:58]:
```suggestion
Los volúmenes efímeros CSI (Container Storage Interface) *deben* ser proporcionados por un controlador de almacenamiento
```

**Comment by gamba47** [comment]
suggestions ok!

Need help on lines 67 and 75

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+Los volúmenes efímeros CSI (Container Storage Interface) *deben* ser proporcionados por un controlador de almacenamiento
+de terceros.
+
+Los volúmenes efímeros genéricos *pueden* ser proporcionados por controladores de almacenamiento
+de terceros, pero tambien puede ser por cualquier otro controlador de almacenamiento que tenga soporte para aprovisionamiento
+dinámico. Algunos drivers CSI estan escritos específicamente para volúmenes
+efímeros CSI y no tienen soporte para aprovisionamiento dinámico: estos 
+no pueden ser usados por volúmenes efímeros genéricos.
+
+```Necesito una revisión a esta frase no quedó bien```
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:67]: @gamba47 puedes eliminar esta linea de la traduccion para poder avanzar con la revision

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+el Pod. Con vinculación inmediata, el planificador de Kubernetes se ve forzado a seleccionar el nodo que tiene
+acceso al volumen una vez que esta disponible.
+
+En términos de la [propiedad de los recursos](/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents),
+un Pod que tiene un almacenamiento efímero genérico es el dueño del `PersistenVolumeClaim`
+que proporciona este almacenamiento efímero. Cuando un Pod es borrado,
+el recolector de basura de Kubernetes borra el PVC, que usualmente
+desencadena el borrado del volumen porque la política de reclamos por defecto de 
+esta clase de almacenamiento es eliminar los volúmenes. Usted puede crear almacenamiento local cuasi-efímero
+usando un `StorageClass` que tenga una politica de recupero de `retención`: el almacenamiento sobrevive al Pod,
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:205]:
```suggestion
usando un `StorageClass` que tenga una política de recuperación de `retención`: el almacenamiento sobrevive al Pod,
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 9]
```diff
@@ -0,0 +1,271 @@
...
+y en este caso se necesita asegurarse que la limpieza de este volumen suceda separadamente.
+
+Mientras estos PVCs existan, ellos pueden ser usados como cualquier otro PVC. En
+particular, estos pueden ser referenciados como fuente de datos para clonar volúmenes o
+instantáneas. El objeto PVC tambien guarda el estado actual del
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:210]:
```suggestion
snapshotting. El objeto PVC también guarda el estado actual del
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 10]
```diff
@@ -0,0 +1,271 @@
...
+interacción con el PVC porque uno no tiene que buscar el mismo una
+vez que el nombre del Pod y del volumen se conocen.
+
+El nombrar de manera determinística tambien introduce un conflicto potencia entre diferentes
+Pods (un Pod "pod-a" con un volumen "scratch" y otro Pod con nombre
+"pod" y un volumen "a-scratch" terminan con el mismo nombre del PVC
+"pod-a-scratch") y entre los Pods y PVCs creados manualmente.
+
+Este tipo de conflictos son detectados: un PVC solamente es usado por un volumen
+efímero si este fue creado por el Pod. Esta verificacion esta basada en la
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:228]:
```suggestion
efímero si este fue creado por el Pod. Esta verificación esta basada en la
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 11]
```diff
@@ -0,0 +1,271 @@
...
+
+{{< caution >}}
+Tenga cuidado cuando nombre Pods y volúmenes dentro del 
+mismo namespace, para que estos conflictos no ocurran.
+{{< /caution >}}
+
+### Seguridad
+
+Habilitar la característica VolumenGenéricoEfímero permite a los usuarios crear
+PVCs indirectamente si ellos pueden crear Pods, aún si ellos no tiene
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:241]:
```suggestion
PVCs indirectamente si ellos pueden crear Pods, aún si ellos no tienen
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 12]
```diff
@@ -0,0 +1,271 @@
...
+permisos para crear PVCs de manera directa. Los administradores del cluster deben
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:242]:
```suggestion
permisos para crear PVCs de manera directa. Los administradores del clúster deben
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 13]
```diff
@@ -0,0 +1,271 @@
...
+ser concientes de esto. Si esto no cumple con el modelo de seguridad, se tienen
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:243]:
```suggestion
ser conscientes de esto. Si esto no cumple con el modelo de seguridad, se tienen
```

File: content/es/docs/concepts/storage/ephemeral-volumes.md [hunk 14]
```diff
@@ -0,0 +1,271 @@
...
+dos opciones:
+- Usar una [Politica de Seguridad en Pods](/docs/concepts/policy/pod-security-policy/) donde la
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/ephemeral-volumes.md:245]:
```suggestion
- Usar una [Política de Seguridad en Pods](/docs/concepts/policy/pod-security-policy/) donde la
```

**Comment by Sea-n** [comment]
Hey @gamba47,

Thanks for purposing this PR. But since you didn't respond for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.

**Comment by electrocucaracha** [comment]
@gamba47 crees que puedas volver a trabajar en este PR, para reabrirlo

**Comment by gamba47** [comment]
No lo quise reabrir todavia, pero estoy con intensiones de resolverlo pronto!

**Pull request closed without merging**